packages/console/src/pages/Connectors/components/ConnectorName/index.tsx

```diff
@@ -41,7 +45,24 @@ const ConnectorName = ({ type, connector, onClickSetup, isShowId = false }: Prop
     <Link to={`/connectors/${connector.id}`} className={styles.link}>
       <ItemPreview
         title={<UnnamedTrans resource={connector.name} />}
-        subtitle={conditional(isShowId && connector.id)}
+        subtitle={
+          <>
+            {type !== ConnectorType.Social && connector.id}
+            {connectors.length > 1 && (
+              <div className={styles.platforms}>
+                {connectors.map(
+                  ({ id, platform }) =>
+                    platform && (
+                      <div key={id} className={styles.platform}>
+                        <ConnectorPlatformIcon platform={platform} />
+                        {t(connectorPlatformLabel[platform])}
+                      </div>
+                    )
+                )}
+              </div>
```

Comment:
just wondering should those be  wrapped up in `type === ConnectorType.Social`?

Reply:
You are right, that's more accurate, updated